### PR TITLE
Webhooks update for GA/hideClassic

### PR DESCRIPTION
**Pull request by slisaasquatch**
## Description of the change

> Description here

## Types of changes

- [ ] Documentation content
- [ ] Page Layout / templates / structure
- [ ] Internal / code cleanup / build system

### CODE REVIEW

**Review by johanventer — APPROVED** [review]

File: content/api/webhooks.md [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
 Interacting with third party APIs like SaaSquatch often suffers from two important problems:
 
-
- - Services not directly responsible for making an API request may still need to know the response
- - Some events, like coupon created events, are not the result of a direct API request
-
+- Services not directly responsible for making an API request may still need to know the response
+- Some events, like coupon created events, are not the result of a direct API request
 
 Webhooks solve these problems by letting you register a URL that we will POST to anytime an event happens in your account. When the event
```
Comment (rebeccachan) [inline content/api/webhooks.md:14]: Can we make sure any call-type words like `POST` are encapsulated in `` like the rest of the formatting? Same with any response codes.

Comment (rebeccachan) [inline content/api/webhooks.md:14]: "When the event" --> "When an event"

File: content/api/webhooks.md [hunk 2]
```diff
@@ -1,37 +1,31 @@
...
 occurs, for example when a vanity coupon code is created for a new user, SaaSquatch creates an event object. This object contains all the relevant
 information, including the type of event and the data associated with that event. SaaSquatch then sends an HTTP POST request with the event object
```
Comment (rebeccachan) [inline content/api/webhooks.md:16]: "including the type of event and the data associated with that event." ---> "including the type of event and the data associated with that event, such as the confirmed value of the vanity coupon." 

Calling out an example previously then neglecting to refer to it isn't very helpful haha!

File: content/api/webhooks.md [hunk 3]
```diff
@@ -1,37 +1,31 @@
...
 to any URLs in your account's webhook settings. You can find a full list of all event types below.
```
Comment (rebeccachan) [inline content/api/webhooks.md:17]: new paragraph before "You can find" ... pls

File: content/api/webhooks.md [hunk 4]
```diff
@@ -236,92 +226,150 @@ events. New event types may be added to the API, so avoid building logic that as
 Sent whenever a new User is created. Note: Users can be created via the REST Api, the Squatch.js javascript library, loading the referral
```
Comment (rebeccachan) [inline content/api/webhooks.md:226]: "REST Api" ---> "REST API"

File: content/api/webhooks.md [hunk 5]
```diff
@@ -331,20 +379,20 @@ Sent whenever a rewards balance is updated. Only works with all `unit` types for
...
 Other rules to consider:
 
- - The `user.reward.balance.changed` webhook only applies to `CREDIT` rewards. `PCT_DISCOUNT`, `FUELTANK` and `INTEGRATION` rewards won't trigger this webhook
- - The creation of a pending reward will not trigger a balance update because it doesn't affect the balance
- - This webhook is per unit balance changed (we don't combine balances in a single webhook)
- - This webhook contains an available balance value calculated after (not exactly at) the time of the trigger
- - No balance update webhook is sent when a user is deleted and any external user balance will remain at its last known value
+- The `user.reward.balance.changed` webhook only applies to `CREDIT` rewards. `PCT_DISCOUNT`, `FUELTANK` and `INTEGRATION` rewards won't trigger this webhook
+- The creation of a pending reward will not trigger a balance update because it doesn't affect the balance
+- This webhook is per unit balance changed (we don't combine balances in a single webhook)
```
Comment (rebeccachan) [inline content/api/webhooks.md:393]: "we don't combine balances in a single webhook" --> "different unit types are not combined into a single webhook"

File: content/api/webhooks.md [hunk 6]
```diff
@@ -504,170 +549,258 @@ Sent whenever a referral is converted.
...
+      }
     }
+  }
 }
 ```
 
-### referral.ended
+### <span class="label">Classic only</span> referral.ended
 
 Sent whenever a referred user, or referral, is cancelled.
```
Comment (rebeccachan) [inline content/api/webhooks.md:755]: This doesn't actually make any sense without context. We can't "cancel" Referred Users--the way this is worded implies that a referral ended webhook is sent if a Referred User is deleted, which is not the case. I know this is an old Payment Provider-only use case, so please indicate that in the description as well.

**Review by rebeccachan — CHANGES_REQUESTED** [review]
I haven't actually had a chance to go over the webhooks page in the docs for more clarity, but that is on the list after this quick review.

**Pull request merged**